recipes/pygeoprocessing/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pygeoprocessing" %}
+{% set version = "2.0.0" %}
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 441cef0679d6fe46fb0f9c14cac78d3db75a097f2596c91b345ca2f40002e4be
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+    host:
+        - python
+        - pip
+        - cython
+        - numpy
+        - setuptools_scm
+    run:
+        - python
+        - rtree
+        - scipy
+        - gdal
+        - shapely
+        - setuptools
+
```

Comment:
If numpy is used for C/C++ extensions, add to the run deps:
```
{{ pin_compatible('numpy') }}
```
Or, as the wheel suggests:
```
numpy >=1.10.1
```

Reply:
Added `numpy >=1.10.1` as suggested, since that's what's specified in `requirements.txt`.  Added in 67ab66f54.

Reply:
I looked at the build log, it is using the numpy C API. You need to drop the version from both host and run and use the following in run:
```
{{ pin_compatible('numpy') }}
```

Please go through: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Ah, very nice!  I was wondering how pinning would work within the conda-forge ecosystem.  Thanks for the reference.  I've patched this in e496d4305.